app/javascript/components/CreateTransaction.jsx

```diff
@@ -105,18 +101,15 @@ class CreateTransaction extends React.Component {
   }
 
   handleSubmit = (event) => {
-    const artist_id = this.props.artist.id;
-    const transactions_route = APIRoutes.transactions.create;
+    const transactions_route = APIRoutes.receipts.create;
     const valid = this.state.formValid;
     const payload = {
-      "type": this.state.transaction.transaction_type,
+      "transaction_type": this.state.transaction.transaction_type,
```

Comment:
should we still have the option to record the type of transaction since rental/purchase is being chosen upon making a request?

Reply:
I think so, since a buyer may change their mind